tests/test_mets.py

```diff
@@ -158,6 +159,46 @@ def test_fromfile_invalid_xlink_href(self):
             metsrw.METSDocument.fromfile(
                 'fixtures/mets_invalid_xlink_hrefs.xml')
 
+    def test__analyze_fptr(self):
```

Comment:
I think the usual convention is to just use one underscore even for "private" methods.